.travis.yml

```diff
@@ -58,16 +50,6 @@ matrix:
       env: TOXENV=nginxroundtrip
 
 
-# Only build pushes to the master branch, PRs, and branches beginning with
```

Comment:
Sure, restored.

It is a pain for contributors though, it means you cannot test your feature branch on the CI *before* creating a PR. I prefer not to submit a PR before it's green. Ideally these restrictions would not apply to forks, but this can be left for now.

Reply:
That's fair. If there's a way to do this in a way that doesn't affect forks, I'd happily accept another PR containing these changes.